src/components/v-menu/v-menu.vue

```diff
@@ -0,0 +1,737 @@
+		const activatorFixed = ref(false);
+		const calculatedTopAuto = ref(0);
+		const resizeTimeout = ref(0);
+		const delays = reactive({
+			openTimeout: undefined as number | undefined,
+			closeTimeout: undefined as number | undefined
+		});
+
+		const dimensions = reactive({
+			activator: reactive({
+				top: 0,
+				left: 0,
+				bottom: 0,
+				right: 0,
+				width: 0,
+				height: 0,
+				offsetTop: 0,
+				scrollHeight: 0,
+				offsetLeft: 0
+			}),
+			content: reactive({
+				top: 0,
+				left: 0,
+				bottom: 0,
+				right: 0,
+				width: 0,
+				height: 0,
+				offsetTop: 0,
+				scrollHeight: 0
+			})
+		});
+		const { width, height } = useWindowSize();
+		watch(width, () => onResize());
+		watch(height, () => onResize());
+		watch(
+			() => props.disabled,
+			(val: boolean) => val && callDeactivate()
+		);
+		watch(
+			() => props.value,
+			(val: boolean) => (isActive.value = !!val)
+		);
+		watch(isActive, (val: boolean) => {
+			!!val !== props.value && context.emit('input', val);
+		});
+		watch(isActive, (val: boolean) => {
+			if (props.disabled) return;
+			val ? callActivate() : callDeactivate();
+		});
+		watch(() => props.positionX, updateDimensions);
+		watch(() => props.positionY, updateDimensions);
+
+		watch(isContentActive, val => (hasJustFocused.value = val));
+
+		onBeforeMount(() => {
+			hasWindow.value = typeof window !== 'undefined';
+		});
+		onMounted(() => {
+			isActive.value && callActivate();
+		});
+
+		/**
+		 *
+		 * (De)Activate
+		 *
+		 **/
+
+		function getActivator(e?: Event): HTMLElement | null {
+			// If we've already fetched the activator, re-use
+			if (activatorElement.value) return activatorElement.value;
+
+			if (e) {
+				// Activated by a click event
+				const activator = (e.currentTarget || e.target) as HTMLElement;
+				activatorElement.value = activator;
+			}
+
+			return activatorElement.value;
+		}
+		function checkActivatorFixed() {
+			if (attach.value !== false) return; // this.attach !== false
+			let el = getActivator();
+			while (el) {
+				if (window.getComputedStyle(el).position === 'fixed') {
+					activatorFixed.value = true;
+					return;
+				}
+				el = el.offsetParent as HTMLElement;
+			}
+			activatorFixed.value = false;
+		}
+		function activate() {
+			// Update coordinates and dimensions of menu
+			// and its activator
+
+			updateDimensions();
+			// Start the transition
+			requestAnimationFrame(() => {
+				// Once transitioning, calculate scroll and top position
+				startTransition().then(() => {
+					if (content.value !== null) {
+						calculatedTopAuto.value = calcTopAuto();
+						props.auto && (content.value.scrollTop = calcScrollPosition());
+					}
+				});
+			});
+		}
+		function callActivate() {
+			if (!hasWindow) return;
+			activate();
+		}
+		function callDeactivate() {
+			isContentActive.value = false;
+		}
+
+		/**
+		 *
+		 * Calc Position
+		 *
+		 **/
+		const calculatedLeft = computed((): string => {
```

Comment:
You can typecast computed's as follows:

```suggestion
		const calculatedLeft = computed<string>(() => {
```